From d7ace6a2f66e96aa2517e2b5090480089240013c Mon Sep 17 00:00:00 2001
From: Celine Chotard <celine.chotard@intel.com>
Date: Fri, 31 May 2013 15:20:43 +0200
Subject: [PORT FROM MAIN] Renaming of ctpscaleht by ctp7160

BZ: 112157

Change-Id: I6a91219d5b45ebee75a020fea1e012ec3f073cf3
Orig-Change-Id: Ie1778787b1b6cc2bab3018afc9ce56046f470811
Signed-off-by: Celine Chotard <celine.chotard@intel.com>
Reviewed-on: http://android.intel.com:8080/111116
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 libbt/conf/intel/ctp7160/bt_vendor.conf    |    5 +++++
 libbt/conf/intel/ctpscaleht/bt_vendor.conf |    5 -----
 2 files changed, 5 insertions(+), 5 deletions(-)
 create mode 100644 libbt/conf/intel/ctp7160/bt_vendor.conf
 delete mode 100644 libbt/conf/intel/ctpscaleht/bt_vendor.conf

diff --git a/libbt/conf/intel/ctp7160/bt_vendor.conf b/libbt/conf/intel/ctp7160/bt_vendor.conf
new file mode 100644
index 0000000..93aa23c
--- /dev/null
+++ b/libbt/conf/intel/ctp7160/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFileRootPath = /etc/firmware/bt/
diff --git a/libbt/conf/intel/ctpscaleht/bt_vendor.conf b/libbt/conf/intel/ctpscaleht/bt_vendor.conf
deleted file mode 100644
index 93aa23c..0000000
--- a/libbt/conf/intel/ctpscaleht/bt_vendor.conf
+++ /dev/null
@@ -1,5 +0,0 @@
-# UART device port where Bluetooth controller is attached
-UartPort = /dev/ttyMFD0
-
-# Firmware patch file location
-FwPatchFileRootPath = /etc/firmware/bt/
-- 
1.7.4.1

